container/docker/handler.go

```diff
@@ -274,6 +291,15 @@ func newDockerContainerHandler(
 		}
 	}
 
+	// retrieve desired labels
+	for _, exposedLabel := range labelWhitelist {
+		labelValue, present := ctnr.Config.Labels[exposedLabel]
+		if !present {
```

Comment:
nit: condense this
if labelValue, present := ctnr.Config.Labels[exposedLabel]; present {
handler.labels[exposedLabel] = labelValue
}